pkg/controller/fileintegrity/fileintegrity_controller.go

```diff
@@ -23,6 +23,143 @@ import (
 
 var log = logf.Log.WithName("controller_fileintegrity")
 
+const (
+	fileIntegrityNamespace     = "openshift-file-integrity"
+	defaultConfDataKey         = "aide.conf"
+	defaultConfigMapName       = "aide-conf"
+	workerDaemonSetName        = "aide-worker"
+	masterDaemonSetName        = "aide-master"
+	aideScriptConfigMapName    = "aide-script"
+	operatorServiceAccountName = "file-integrity-operator"
+	aideScriptPath             = "/scripts/aide.sh"
+)
+
+var aideScript = `
+    #!/bin/sh
+    if test ! -f /hostroot/etc/kubernetes/aide.db.gz; then
```

Comment:
Is there a reason for this being in /etc/kubernetes ? maybe we should move this elsewhere to be more specific to this operator. we could have an init container create /etc/aide on the host if needed... or something of the sort.

Reply:
Filed https://github.com/mrogers950/file-integrity-operator/issues/4 to continue this